tt_test.go

```diff
@@ -0,0 +1,524 @@
+	})
+}
+
+func (m MockTypeToTTConvFactory) GetStringConverter() tupleconv.Converter[any, any] {
+	return tupleconv.MakeFuncConverter(func(s any) (any, error) {
+		return "string", nil
+	})
+}
+
+func (m MockTypeToTTConvFactory) GetUnsignedConverter() tupleconv.Converter[any, any] {
+	return tupleconv.MakeFuncConverter(func(s any) (any, error) {
+		return "unsigned", nil
+	})
+}
+
+func (m MockTypeToTTConvFactory) GetDatetimeConverter() tupleconv.Converter[any, any] {
+	return tupleconv.MakeFuncConverter(func(s any) (any, error) {
+		return "datetime", nil
+	})
+}
+
+func (m MockTypeToTTConvFactory) GetUUIDConverter() tupleconv.Converter[any, any] {
+	return tupleconv.MakeFuncConverter(func(s any) (any, error) {
+		return "uuid", nil
+	})
+}
+
+func (m MockTypeToTTConvFactory) GetMapConverter() tupleconv.Converter[any, any] {
+	return tupleconv.MakeFuncConverter(func(s any) (any, error) {
+		return "map", nil
+	})
+}
+
+func (m MockTypeToTTConvFactory) GetArrayConverter() tupleconv.Converter[any, any] {
+	return tupleconv.MakeFuncConverter(func(s any) (any, error) {
+		return "array", nil
+	})
+}
+
+func (m MockTypeToTTConvFactory) GetVarbinaryConverter() tupleconv.Converter[any, any] {
+	return tupleconv.MakeFuncConverter(func(s any) (any, error) {
+		return "varbinary", nil
+	})
+}
+
+func (m MockTypeToTTConvFactory) GetDoubleConverter() tupleconv.Converter[any, any] {
+	return tupleconv.MakeFuncConverter(func(s any) (any, error) {
+		return "double", nil
+	})
+}
+
+func (m MockTypeToTTConvFactory) GetDecimalConverter() tupleconv.Converter[any, any] {
+	return tupleconv.MakeFuncConverter(func(s any) (any, error) {
+		return "decimal", nil
+	})
+}
+
+func (m MockTypeToTTConvFactory) GetIntegerConverter() tupleconv.Converter[any, any] {
+	return tupleconv.MakeFuncConverter(func(s any) (any, error) {
+		return "integer", nil
+	})
+}
+
+func (m MockTypeToTTConvFactory) GetNumberConverter() tupleconv.Converter[any, any] {
+	return tupleconv.MakeFuncConverter(func(s any) (any, error) {
+		return "number", nil
+	})
+}
+
+func (m MockTypeToTTConvFactory) GetAnyConverter() tupleconv.Converter[any, any] {
+	return tupleconv.MakeFuncConverter(func(s any) (any, error) {
+		return "any", nil
+	})
+}
+
+func (m MockTypeToTTConvFactory) GetScalarConverter() tupleconv.Converter[any, any] {
+	return tupleconv.MakeFuncConverter(func(s any) (any, error) {
+		return "scalar", nil
+	})
+}
+
+func (m MockTypeToTTConvFactory) GetIntervalConverter() tupleconv.Converter[any, any] {
+	return tupleconv.MakeFuncConverter(func(s any) (any, error) {
+		return "interval", nil
+	})
+}
+
+func (m MockTypeToTTConvFactory) MakeNullableConverter(
+	c tupleconv.Converter[any, any]) tupleconv.Converter[any, any] {
+	return tupleconv.MakeFuncConverter(func(s any) (any, error) {
+		return "null", nil
+	})
+}
+
+var _ tupleconv.TTConvFactory[any] = (*MockTypeToTTConvFactory)(nil)
+
+func TestMakeTypeToTTConverters(t *testing.T) {
+	t.Run("basic", func(t *testing.T) {
```

Comment:
I don't quite understand why the test is written like that. This can be easily replaced with `TestMakeTypeToTTConverters_basic` and it will be 1 nesting level less.

Reply:
Reworked. It was just a way to group the tests.